packages/dds/tree/src/test/domains/json/twitter.ts

```diff
@@ -0,0 +1,634 @@
+        profile_banner_url: "https://pbs.twimg.com/profile_banners/2699365116/1406936481",
+        profile_link_color: getRandomEnglishString(random, true, 6, 6),
+        profile_sidebar_border_color: getRandomEnglishString(random, true, 6, 6),
+        profile_sidebar_fill_color: getRandomEnglishString(random, true, 6, 6),
+        profile_text_color: getRandomEnglishString(random, true, 6, 6),
+        profile_use_background_image: random.bool(),
+        default_profile: random.bool(),
+        default_profile_image: random.bool(),
+        following: random.bool(),
+        follow_request_sent: random.bool(),
+        notifications: random.bool(),
+    };
+    if (shouldAddUrlUrlsEntity) {
+        user.entities.url = {
+            urls: [
+                {
+                    url: "http://t.co/V4oyL0xtZk",
+                    expanded_url: "http://astore.amazon.co.jp/furniturewood-22",
+                    display_url: "astore.amazon.co.jp/furniturewood-…",
+                    indices: [
+                        random.integer(0, 199),
+                        random.integer(0, 199),
+                    ],
+                },
+            ],
+        };
+    }
+    if (shouldAddDescriptionUrlsEntity) {
+        user.entities.description.urls.push(
+            {
+                url: "http://t.co/8E91tqoeKX",
+                expanded_url: "http://ameblo.jp/2no38mae/",
+                display_url: "ameblo.jp/2no38mae/",
+                indices: [
+                    random.integer(0, 199),
+                    random.integer(0, 199),
+                ],
+            },
+        );
+    }
+
+    return user;
+}
+/* eslint-enable */
+
+function getBasicJapaneseAlphabetString() {
+    // Japanese Hiragana
+    return createAlphabetFromUnicodeRange(0x3041, 0x3096) +
+        // Japanese Katakana (Full Width)
+        createAlphabetFromUnicodeRange(0x30A0, 0x30FF) +
+        // Japanese Kanji Alphabet (CJK Unified Ideographs)
+        createAlphabetFromUnicodeRange(0x3400, 0x4DB5) +
+        createAlphabetFromUnicodeRange(0x4E00, 0x9FCB) +
+        createAlphabetFromUnicodeRange(0xF900, 0xFA6A);
+}
+
+// This is specifically formatted like the twitter json dates
+// (<3-letter-weekday> MMM DD HH:MM:SS <4-digit-TimezoneOffset> YYYY)
+function getRandomDateString(random = makeRandom(), start: Date, end: Date) {
+    const dateS = new Date(random.integer(+start, +end)).toString();
+    return `${dateS.substring(0, 10)} ${dateS.substring(16, 24)} ` +
+        `${dateS.substring(28, 33)} ${dateS.substring(11, 15)}`;
+}
+
+// Source for unicode ranges:
+// https://stackoverflow.com/questions/19899554/
+// unicode-range-for-japanese#:~:text=To%20summarize%20the%20ranges%3A,Katakana%20(%2030a0%20%2D%2030ff)
+// or more direct source:
+// http://www.localizingjapan.com/blog/2012/01/20/regular-expressions-for-japanese-text/
+export function isJapanese(ch: string) {
```

Comment:
Around this point, these begin to look like helpers for parsing/analyzing the original JSON.  These should probably live somewhere else, possibly jsonGeneratorUtils.ts?

Reply:
The reason I left it in twitter.ts is because it was specific to analyzing data within the twitter.json and I didn't see any foreseeable shared use case.